### DISCUSSION TITLE
v2.9.5-3dfx

### DISCUSSION THREAD

**Discussion by xoureldeen** [xoureldeen/Vectras-VM-Android#180] [General]
## Changes
- Bring back 3dfx support.
- Enhance app execution.
- Added some Linux programs in X11 display.
- Added Alpine Linux Xfce4 (X11).
- Russian language by [@OFGING](https://github.com/OFGING).
> [!WARNING]
> 3dfx only available for 64bit devices.

## Iso wrappers 3dfx (only 64bit)
- for windows vm: [3dfx-wrappers-2.9.5.iso](https://github.com/xoureldeen/Vectras-VM-Android/raw/refs/heads/master/3dfx/3dfx-wrappers-2.9.5.iso)

## Bootstraps
- [Arm64-v8a](https://github.com/xoureldeen/Vectras-VM-Android/releases/download/v2.9.3/vectras-vm-arm64-v8a.tar.gz) 
- [X86_64](https://github.com/xoureldeen/Vectras-VM-Android/releases/download/v2.9.3/vectras-vm-x86_64.tar.gz) 
- armeabi-v7a: no need.
- x86: no need.

<hr /><em>This discussion was created from the release <a href='https://github.com/xoureldeen/Vectras-VM-Android/releases/tag/v2.9.5'>v2.9.5-3dfx</a>.</em>

**Comment by GmoCRojY** [comment]
the app itself works that features glitches like sound and other features combining TEMUX and Create Qemu x86-64 command that I'm testing on real phones... there is a lack of bookstores
